conf/config.go

```diff
@@ -88,6 +90,10 @@ type MenderConfig struct {
 	RootfsScriptsPath   string
 }
 
+type DbusConfig struct {
```

Comment:
Nitpick: Inconsistent naming, `DbusConfig` should be `DBusConfig` to follow the style of the rest.

Discovered while regex searching with case sensitivity! :-)

Reply:
Fixed